pyspark/bigdl/optim/optimizer.py

```diff
@@ -400,8 +444,10 @@ def set_validation(self, batch_size, val_rdd, trigger, val_method=["Top1Accuracy
         :param trigger: validation interval
         :param val_method: the ValidationMethod to use,e.g. "Top1Accuracy", "Top5Accuracy", "Loss"
         """
+        if val_method is None:
+            val_method = [Top1Accuracy()]
```

Comment:
Looks like if we set val_method's default value in the parameter list, code is more clean. Is there a reason why we set none in the parameter list and update it in the implementation.